### DISCUSSION TITLE
Xarrow is going out of the modal. Z-index is not working either.

### DISCUSSION THREAD

**Discussion by tejab92** [Eliav2/react-xarrows#69] [Q&A]
![image](https://user-images.githubusercontent.com/82674602/117241883-f3d87800-ade8-11eb-848a-448801bc57b3.png)
![image](https://user-images.githubusercontent.com/82674602/117241887-f6d36880-ade8-11eb-9b15-f41816ef8f03.png)

I would like the arrow to be hidden when the tree is scrolled, so that anything beyond the modal will be hidden. But in the second screenshot, you can observe the line is going beyond the modal. I tried setting z-index but doesn't work. Any help or thoughts would be appreciated. @Eliav2

**Comment by Eliav2** [comment]
please copy the entire tree component and produce a minimal code sandbox which demonstrates this issue so I could debug it.
